dcap-provider/Cargo.toml

```diff
@@ -23,7 +23,8 @@ DCAP: DataCenter Attestation Primitives
 QL: Quoting Library
 """
 repository = "https://github.com/fortanix/rust-sgx"
-documentation = "https://docs.rs/dcap-provider/"
+documentation = "https://edp.fortanix.com/docs/"
```

Comment:
documentation = "https://edp.fortanix.com/docs/api/dcap-provider" ?

Reply:
No, this crate doesn't have any useful documentation. It doesn't have any documentation right now but it's not going to live there